docs/guide/building-a-basic-scene.md

```diff
@@ -44,9 +60,16 @@ To translate, rotate, and scale the box, we can plug in the [position][position]
 </a-scene>
 ```
 
+The example above (assuming we are positioned on the origin looking down the
+negative Z-axis) will translate the box left/up/back, rotate the box to the
+right, stretches the box left-and-right and back-and-front, and shrinks the box
+up-and-down:
+
 ## Applying a Texture to the Box
 
-The box doesn't have to be just a flat color. We can wrap a texture around the box with an image or video using `src`. To make sure the color does not mix with the texture, we set the color to white:
+We can apply an image texture to the box with an image or video using the `src`
+attribute. To make sure the color does not mix with the texture, we set the
+back color to white:
 
 ```html
```

Comment:
any idea why [the syntax highlighting](aframe.io/docs/master/guide/building-a-basic-scene.html#applying-a-texture-to-the-box) doesn't seem to be working with the newlines?

![image](https://cloud.githubusercontent.com/assets/203725/16635755/9ea02cea-4388-11e6-98e5-d11a1d7cea10.png)